diff -crB original/shorewall/interfaces /etc/shorewall/interfaces
*** original/shorewall/interfaces	2009-06-11 16:12:35.000000000 -0700
--- /etc/shorewall/interfaces	2009-08-16 14:15:31.000000000 -0700
***************
*** 8,11 ****
--- 8,12 ----
  #
  ###############################################################################
  #ZONE	INTERFACE	BROADCAST	OPTIONS
+ net	eth0	-	routefilter,blacklist,tcpflags
  #LAST LINE -- ADD YOUR ENTRIES BEFORE THIS ONE -- DO NOT REMOVE
diff -crB original/shorewall/policy /etc/shorewall/policy
*** original/shorewall/policy	2009-06-11 16:12:35.000000000 -0700
--- /etc/shorewall/policy	2009-08-16 14:16:23.000000000 -0700
***************
*** 9,12 ****
--- 9,15 ----
  ###############################################################################
  #SOURCE	DEST	POLICY		LOG	LIMIT:		CONNLIMIT:
  #				LEVEL	BURST		MASK
+ $FW	net	ACCEPT
+ net	all	DROP
+ all	all	REJECT
  #LAST LINE -- DO NOT REMOVE
diff -crB original/shorewall/routestopped /etc/shorewall/routestopped
*** original/shorewall/routestopped	2009-06-11 16:12:35.000000000 -0700
--- /etc/shorewall/routestopped	2009-08-16 14:17:58.000000000 -0700
***************
*** 12,15 ****
--- 12,16 ----
  ###############################################################################
  #INTERFACE	HOST(S)			OPTIONS		PROTO	DEST	SOURCE
  #								PORT(S)	PORT(S)
+ eth0	-	routeback,source,dest,critical
  #LAST LINE -- ADD YOUR ENTRIES BEFORE THIS ONE -- DO NOT REMOVE
diff -crB original/shorewall/rules /etc/shorewall/rules
*** original/shorewall/rules	2009-06-11 16:12:35.000000000 -0700
--- /etc/shorewall/rules	2009-08-16 14:17:39.000000000 -0700
***************
*** 12,15 ****
--- 12,17 ----
  #SECTION ESTABLISHED
  #SECTION RELATED
  SECTION NEW
+ ACCEPT	net	$FW	tcp	ssh,smtp,http,https,10001,8080,225
+ ACCEPT	net	$FW	icmp
  #LAST LINE -- ADD YOUR ENTRIES BEFORE THIS ONE -- DO NOT REMOVE
diff -crB original/shorewall/zones /etc/shorewall/zones
*** original/shorewall/zones	2009-06-11 16:12:35.000000000 -0700
--- /etc/shorewall/zones	2009-08-16 14:13:55.000000000 -0700
***************
*** 10,13 ****
--- 10,14 ----
  #ZONE	TYPE		OPTIONS		IN			OUT
  #					OPTIONS			OPTIONS
  fw	firewall
+ net	ipv4				#
  #LAST LINE - ADD YOUR ENTRIES ABOVE THIS ONE - DO NOT REMOVE
